src/DataReader/Cache/CachedCount.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Cycle\DataReader\Cache;
+
+use Countable;
+
+final class CachedCount
```

Comment:
Same about this one. It doesn't add much and removing it will make things simpler.

Reply:
OK. Got the idea.